plugins/lando-services/php/5.4-apache/Dockerfile

```diff
@@ -76,3 +76,10 @@ RUN apt-get update && apt-get install -y \
   && apt-get -y autoclean \
   && apt-get -y autoremove \
   && rm -rf /var/lib/apt/lists/* && rm -rf && rm -rf /var/lib/cache/* && rm -rf /var/lib/log/* && rm -rf /tmp/*
+
+# Install Blackfire agent
```

Comment:
You aren't actually installing the agent here - this is the probe.  The agent is inside the "blackfire" container.  Ditto for all of the Dockerfile modifications below.